packages/jest-util/src/shuffleArray.ts

```diff
@@ -0,0 +1,38 @@
+// Mulberry 32 taken from
+// https://stackoverflow.com/questions/521295/seeding-the-random-number-generator-in-javascript
+export function mulberry32(seed: number): {next: () => number} {
```

Comment:
@SimenB instead of coding a dedicated random generator within jest, we may leverage existing libraries such as pure-rand (the library backing fast-check) or any other one?

Reply:
It comes with a pretty optimized and random ring called xoroshiro (the exact same ring as math random of V8 but in a seeded version)

Reply:
Sounds reasonable 👍

Reply:
Just seen the head of the PR leveraged prando which has far less traction than pure-rand 🤔 (pure-rand downloads being mostly due to fast-check)

Reply:
tbh I found the first rng package that seemed to fit the needs. It won't be hard to swap to pure-rand.
As you're the owner of pure-rand and a contributor to jest it makes a lot of sense to use pure-rand instead.

I'll make the change after I create a new PR just exposing the seed argument.